### ISSUE TITLE
Warning: not using a pretrained token vae

### ISSUE THREAD

**Issue by SMZ1834521663** [brepdiff/brepdiff#5]
I haven't seen the pre trained model of uvuvae, how can I get it

**Comment by 96lives** [comment]
I dont mean that its not used. rather you wont need any weights to run it because its not actually a vae

**Comment by SMZ1834521663** [comment]
Can you provide the loss at the beginning and end of the first round of diffusion training(ABC dataset). Thanks

**Comment by 96lives** [comment]
This is for ABC 50 dataset 

<img width="1147" height="378" alt="Image" src="https://github.com/user-attachments/assets/3b863d48-db16-440a-b271-dadaffe1363d" />

This is for DeepCAD 30 dataset

<img width="1147" height="378" alt="Image" src="https://github.com/user-attachments/assets/54ca84c9-ea63-48c4-8dff-a57d26b29cbb" />

I suggest you run DeepCAD dataset for sanity check.
The loss is picky, but it should be fine. It does not happen often, but If the loss, explodes, I suggest you run it again.

**Comment by SMZ1834521663** [comment]
<img width="1287" height="855" alt="Image" src="https://github.com/user-attachments/assets/2ba152d1-409c-4c25-924b-fd0674409c2d" />
Hello, I have a question here
1.param grid_mask: Why use three masks to identify whether a point is on a surface? In my impression, the output mask using "occwl" is B x n_primes x n_grid x n_grid x 1

**Comment by 96lives** [comment]
Could depend on hardware, but I suspect should be less then 10 seconds when it succeeds. Please refer to the paper, as there is the statistics regarding the timings in table 2.

**Comment by 96lives** [comment]
We use the first 3,000 successfully post-processed Brep files for testing. Since the success rate of the postprocessing is not idea, I usually generate 10,000 samples in the diffusion pipeline and post-process until we have 3,000 samples.

**Comment by SMZ1834521663** [comment]
“Specifically, we use 83,205 training
samples for DeepCAD (with a maximum of30 faces per sample) and
259,593 training samples”
May I ask if this refers to the pure training part, which is around 90% of the values.

**Comment by SMZ1834521663** [comment]
May I ask if your ABC dataset uses abc_data_stlit_6bit.pkl and filters for 0-50 surface ranges? After removing duplicates, there are still 300000 data points, which is quite strange
